workloads/kube-burner/workloads/networkpolicy/README.md

```diff
@@ -0,0 +1,33 @@
+
+## Networkpolicy 
+
+With the help of [networkpolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) object we can control traffic flow at the IP address or port level in Kubernetes
+
+## How to run networkpolicy tests?
+
+The environmental variables and steps to kick off this test can be found [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/README.md#kube-burner-e2e-benchmarks)
+
+## What are the test cases that this workload can currently run?
+
+A networkpolicy can come in various shapes and sizes. Allow traffic from a specific namespace, Deny traffic from a specific pod IP, Deny all traffic, ...
+Hence we have come up with a few test cases which try to cover most of them. They are as follows.
+
+### Case 2
```

Comment:
Perhaps add all cases to the list and explain them and then call out the ones specifically implemented - so it makes sense when you say case2 and case3?